src/main/java/com/matera/account/account/Account.java

```diff
@@ -64,11 +68,11 @@ public void setClientId(UUID clientId) {
     }
 
     @Override
-    public Integer getAccountTypeId() {
+    public AccountType getAccountTypeId() {
```

Comment:
same on the get/set methods, changing to getAccountType/setAccountType is fine now